runtime/src/main/java/org/corfudb/runtime/collections/CorfuTable.java

```diff
@@ -428,8 +448,8 @@ protected void undoPut(CorfuTable<K, V> table, V undoRecord, K key, V value) {
      * @param undoRecord    The undo record generated by undoRemoveRecord
      */
     @DontInstrument
-     void undoPutAll(CorfuTable<K, V> table, Map<K,V> undoRecord,
-                            Map<? extends K, ? extends V> m) {
+    void undoPutAll(CorfuTable<K, V> table, Map<K,V> undoRecord,
```

Comment:
This can actually be omitted for `ICorfuVersionPolicy.MONOTONIC`

Reply:
Ok.